packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -115,7 +123,58 @@ export default Rule.Atomic.of<Page, Element>({
           });
 
           for (const child of children) {
-            yield* visit(child, container);
+            gather(child, container);
+          }
+        }
+
+        function* getApplicableLinks(): Iterable<Element> {
+          // Check if foreground is the same with the parent <p> element
+          const hasDifferentForeground = (
+            element1: Element<string>,
+            element2: Element<string>
+          ): boolean =>
+            getForeground(element1, device)
```

Comment:
Here I'm missing the `context` and it will fall back to the default defined in `get-colors`, is that expected?